src/caf/toolkit/cost_utils.py

```diff
@@ -182,21 +181,26 @@ def __eq__(self, other):
         """Check if two items are the same."""
         if not isinstance(other, CostDistribution):
             return False
-        return (self.df == other.df).values.all()
+        return (self.__df == other.df).values.all()
 
     def copy(self) -> CostDistribution:
         """Create a copy of this instance."""
         return copy.copy(self)
 
+    @property
+    def df(self) -> pd.DataFrame:
+        """A Pandas DataFrame containing the class data."""
+        return self.__df
```

Comment:
Do we want to allow people to access the dataframe directly or should this return a shallow copy?